imagetagger/imagetagger/images/models.py

```diff
@@ -43,7 +43,17 @@ class Meta:
     path = models.CharField(max_length=100, unique=True, null=True)
     name = models.CharField(max_length=100)
     location = models.CharField(max_length=100, null=True, blank=True)
-    description = models.TextField(max_length=1000, null=True, blank=True)
+    """
```

Comment:
Move rendering into a model method here as well. The safety warning should be dispensable in that case as the rendering should not be handled anywhere else.